multi-node/aws/pkg/cluster/config.go

```diff
@@ -4,15 +4,32 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
+	"log"
+	"net"
 	"net/url"
 
 	"gopkg.in/yaml.v2"
 )
 
+const (
+	kubernetesCIDR      = "10.0.0.0/8"
```

Comment:
i agree that this should be a parameter. I was under the impression that the whole cluster had to be inside a /8 address space. For instance, I don't think you can have node ips in 10.0.0.0/8 and service ips in 172.0.0.0/8. if you can, then we need to ditch this configuration directive.